### PR TITLE
adding jenkins/server urls to servers

### PR DESCRIPTION
**Pull request by orenlevitzky-fiverr**

### CODE REVIEW

File: public/scripts/directives/server/directive.js [hunk 1]
```diff
@@ -75,6 +75,14 @@ devflowApp.directive('server', ['userService', 'serverService', 'ngDialog', func
                 window.open(server.server_url, '_blank');
             }
 
+            $scope.hasServerUrl = function(server) {
```
Comment (ArkadyFukzon) [inline public/scripts/directives/server/directive.js]: @orenlevitzky-fiverr this is redundant - just do ng-if="server.server_url"

File: public/scripts/directives/server/directive.js [hunk 2]
```diff
@@ -75,6 +75,14 @@ devflowApp.directive('server', ['userService', 'serverService', 'ngDialog', func
...
+                return server.server_url != null;
+            }
+
+            $scope.hasDeployUrl = function(server) {
```
Comment (ArkadyFukzon) [inline public/scripts/directives/server/directive.js]: same as above

File: public/scripts/directives/server/view.html [hunk 3]
```diff
@@ -31,5 +33,8 @@
     </ul>
 
     <a ng-click="navigateToServer(server)"><h2 ng-class="getState(server)" ng-if="isOnDemand(server)">{{server.server_url.replace('http://', '')}}</h2></a>
-    <h2 ng-class="getState(server)" ng-if="!isOnDemand(server)">{{server.name}}</h2>
+    <h2 ng-class="getState(server)" ng-if="!isOnDemand(server)">
+        <span ng-if="hasServerUrl(server)" class="cursor-pointer" ng-click="navigateToServer(server)">{{server.name}}</span>
```
Comment (ArkadyFukzon) [inline public/scripts/directives/server/view.html]: It's a link - use the 'a' tag, that way you won't need that nasty css hack.

**Review by ArkadyFukzon — CHANGES_REQUESTED** [review]

**Comment by orenlevitzky-fiverr** [comment]
@ArkadyFukzon  Fixed

**Comment by orenlevitzky-fiverr** [comment]
@ArkadyFukzon But please dont merge...

**Comment by ArkadyFukzon** [comment]
dont worry I dont have the permissions :)

**Pull request closed without merging**